src/Renderer/OBB.js

```diff
@@ -75,8 +75,10 @@ class OBB extends THREE.Object3D {
         this.z.scale = elevation.scale > 0 ? elevation.scale : this.z.scale;
         this.z.delta = Math.abs(this.z.max - this.z.min) * this.z.scale;
 
-        this.box3D.min.z = this.natBox.min.z + this.z.min * this.z.scale;
-        this.box3D.max.z = this.natBox.max.z + this.z.max * this.z.scale;
+        const geoidHeight = elevation.geoidHeight || 0;
```

Comment:
Could you add `elevation.geoidHeight` to the documentation please ?
Also, would setting `geoidHeigt` as an `OBB` property be of any interest ?